abb_irb6600_support/launch/robot_interface_download_irb6600_225_255.launch

```diff
@@ -0,0 +1,20 @@
+<!--
+  Manipulator specific version of abb_driver's 'robot_interface.launch'.
+
+  Defaults provided for IRB 6600:
+   - J23_coupled = false
```

Comment:
I meant that it might be good to be specific about which variant this file is providing defaults for (so instead of just `IRB 6600`, `Defaults provided for IRB 6600 - 225/2.55`).